README.md

```diff
@@ -81,28 +80,35 @@ Logging from the following services is supported for both cases:
       ])
     }
 
+## Terraform Versions
+
+Terraform 0.12. Pin module version to ~> 4.x Submit pull-requests to master branch.
+
+Terraform 0.11. Pin module version to ~> 3.5.0. Submit pull-requests to terraform011 branch.
```

Comment:
For some reason I thought 3.5.0 was already tf12. But I'm ok with this.

Reply:
The module currently supports both. v.3.4.0 added support for tf 11 again.